docs/native-components-android.md

```diff
@@ -827,7 +827,7 @@ II. Then implement custom View calling the `create` method:
 
 ```jsx title="MyView.jsx"
 import React, { useEffect, useRef } from 'react';
-import { UIManager, findNodeHandle } from 'react-native';
+import { PixelRatio, UIManager, findNodeHandle } from 'react-native';
```

Comment:
```suggestion
import {
  PixelRatio,
  UIManager,
  findNodeHandle
} from 'react-native';
```

Thank you for the example fix @Agagnon01! 👍 

This is the change which lint expects. Please correct that and PR should be ready to merge.